src/app/static/src/app/components/adr/ADRKey.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="row mb-3">
+        <div class="col-8">
+            <div class="d-flex">
+                <label for="key"
+                       class="font-weight-bold align-self-stretch"
+                       >ADR API Key: </label>
+                <div class="align-self-stretch pl-2">
+                    <div v-if="!editing">
+                        <span class="pr-2">{{keyText}}</span>
+                        <a href="#" v-if="!key" @click="edit">add</a>
+                        <a href="#" v-if="key" @click="edit">edit</a>
+                        <span v-if="key">/</span>
+                        <a href="#" v-if="key" @click="remove">remove</a>
+                    </div>
+                    <div class="input-group"
+                         style="margin-top: -11px; min-width: 390px"
```

Comment:
Is there a reason for bespoke width rather than bootstrap class? Is this the right size for showing real ADR key?

Reply:
The margin is to bottom align the input with the label. The width is the min width needed for a real ADR key.